dist/index.html

```diff
@@ -1,24 +1,24 @@
-<!DOCTYPE html>
-<html>
-  <head>
-    <meta charset="utf-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <meta name="viewport" content="width=device-width,initial-scale=1" />
-    <title>Output Management</title>
-    <script defer="defer" src="runtime.bundle.js"></script>
-    <script defer="defer" src="index.bundle.js"></script>
-  </head>
-  <body>
-    <div class="container">
-      <div class="list">
-        <h2>To-do list</h2>
-        <div class="row">
-          <input id="input-box" placeholder="add to your list" />
-          <button onclick="addToList()">Add</button>
-        </div>
-        <ul id="list-container"></ul>
-        <button onclick="removeList()" class="removeButton">clear list</button>
-      </div>
-    </div>
-  </body>
-</html>
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width,initial-scale=1" />
+    <title>Output Management</title>
+    <script defer="defer" src="runtime.bundle.js"></script>
+    <script defer="defer" src="index.bundle.js"></script>
+  </head>
+  <body>
+    <div class="container">
+      <div class="list">
+        <h2>To-do list</h2>
+        <div class="row">
+          <input id="input-box" placeholder="add to your list" />
+          <button onclick="addToList()">Add</button>
+        </div>
+        <ul id="list-container"></ul>
+        <button onclick="removeList()" class="removeButton">clear list</button>

```

Comment:
- Please make the description of the button more descriptive, as from the requirement it is intended to clear all task that are marked as completed and not clear all tasks. Please use the appropriate text as in the design, so it looks more descriptive.

-  **Text on the Design**

![image](https://github.com/LethaboMot/weppack2/assets/58771507/1d5918e6-0f02-4387-88b3-106848305e61)